.github/ci-scripts/bash/functions-for-build

```diff
@@ -20,6 +20,24 @@ aws_configure_tessellops_artifacts() {
   aws configure set region ap-south-1
 }
 
+bash_build() {
+  set -e
+  DIR="${1}"
+  export PATH=$PATH:/home/github/.local/bin
+  ./$DIR/$DIR.sh
```

Comment:
Is the build logic so different that we cannot put it at top level (under script or Makefile)?
Even if its required to have separate build file per dir, lets name it as `build.sh`

Reply:
We can have it at top level, it's a simple script but just to give an high level idea that this script with generate this script it's kept inside a dir. We can have it as `build.sh`